### ISSUE TITLE
Wiring RFM95W

### ISSUE THREAD

**Issue by thecockroach** [CongducPham/LMIC_low_power#1]
Hi, I don't understend how you connect the module to Arduino Pro Mini, 

Here you use pin 9 for control DI0x pins of RFM95:

```
const lmic_pinmap lmic_pins = {
  .nss = 10,
  .rxtx = LMIC_UNUSED_PIN,
  .rst = 8,
  .dio = {2, 7, 9},
};

```

And in the example of reading temp you're using this pins 9 or 2 too. 
Do you have an schematic of the circuit? Tnks

**Comment by CongducPham** [comment]
Yes, sorry, actually you have to use for the temperature sensor pins those that are not used for the LMIC stack. As the LMIC pins depends on the board you are using, you have to change (comment) according to your configuration. So if:

const lmic_pinmap lmic_pins = {
  .nss = 10,
  .rxtx = LMIC_UNUSED_PIN,
  .rst = 8,
  .dio = {2, 7, 9},
};

then you can chose for instance pin 3 to power the temperature sensor:

#define PIN_POWER 3

regards,

**Comment by CongducPham** [comment]
I've updated the example to use pin 6 to power the temperature sensor and add some comments to better explain the configuration steps.

**Comment by thecockroach** [comment]
Thank you.
I am using this because we migrated from modules to test the RFM95W consumption, we were using the 433 MHz Ra-01 with which we got 2.7 uA, I was using the Arduino_LoRa_Generic_DHT example.

When I went from the same project to the RFM95W using Arduino_LoRa_Generic_DHT I could not lower the 160 uA !, I have my suspicions that the module is not taking the sleep mode, the function setSleepMode () says that everything was ok (state = 0) but there is no difference of consumption after calling this function.

In the Ra-01 module I only have connected the SPI pins of the form: NSS D10, MOSI D11, MISO D12, SCK D13.

In the module RFM95W I have connected the SPI pins of the form: NSS D10, MOSI D11, MISO D12, SCK D13, DIO0 D2, DIO1 D3, DIO2 D5, RESET D4.

So far comparing the two modules it seems that the Ra-01 gives fewer complications, both to handle the low consumption, as libraries (already adapts to the sx1272 project) and has PA_BOOST, 
the RFM95W is using many pins and it is not possible to configure the programs of the project well.

Sorry for my English, thank you very much.
